src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -916,7 +916,7 @@ static void _updateTrimpath(TVG_UNUSED LottieGroup* parent, LottieObject** child
         end = (length * end) + pbegin;
     }
 
-    P(ctx->propagator)->strokeTrim(begin, end, trimpath->type == LottieTrimpath::Type::Individual ? true : false);
+    ctx->propagator->strokeTrim(begin, end, trimpath->type == LottieTrimpath::Type::Simultaneous);
```

Comment:
Please compare size with [text+data+bss+dec],  changing is not good...

before:
![image](https://github.com/thorvg/thorvg/assets/3711518/15c942bb-1082-405c-aa42-74d18a267f82)
after:
![image](https://github.com/thorvg/thorvg/assets/3711518/71fceac3-34ac-4206-8fcd-b49e6e77c562)

Reply:
@hermet am I doing something wrong? def settings, no changes
<img width="482" alt="Zrzut ekranu 2024-05-31 o 12 01 29" src="https://github.com/thorvg/thorvg/assets/67589014/b71910d6-e8a9-4a1a-8332-823f62b3b54c">

Reply:
Please try $strip the binary before $size

Reply:
I got the whole list of symbols that can not be stripped, no changes in the size. I'll try to find out later why it doesn;t work, for now I restored the org code.